[Haproxy] Update Log Collection instructions (#5774)
DataDog/integrations-core: haproxy/README.md

_Available for Agent versions >6.0_

> By default Haproxy will send logs over UDP to port 514. The Agent can listen for these logs on this port, however, binding to a port number under 1024 requires elevated permissions. You can follow the instructions below to set this up, you can also  alternatively use a different port and skip step 3.

Reviewer: ```suggestion
By default Haproxy sends logs over UDP to port 514. The Agent can listen for these logs on this port, however, binding to a port number under 1024 requires elevated permissions. Follow the instructions below to set this up. Alternatively, you can use a different port and skip step 3.
```

    Change the `service` parameter value and configure it for your environment. See the [sample haproxy.d/conf.yaml][5] for all available configuration options.

> 3. Grant access to this port using the `setcap` command:

Reviewer: ```suggestion
3. Grant access to port 514 using the `setcap` command:
```

    ```bash
    sudo setcap CAP_NET_BIND_SERVICE=+ep /opt/datadog-agent/bin/agent/agent
    ```
    

>     You can verify this by running the `getcap` command:

Reviewer: ```suggestion
    Verify the setup is correct by running the `getcap` command:
```

    
    ```bash
    sudo getcap /opt/datadog-agent/bin/agent/agent
    ```
    

>     With the expeced output:

Reviewer: ```suggestion
    With the expected output:
```

    ```bash
    /opt/datadog-agent/bin/agent/agent = cap_net_bind_service+ep
    ```
    

>     **Note:** This `setcap` command will need to be re-ran following Agent upgrades.

Reviewer: ```suggestion
    **Note:** Re-run this `setcap` command every time you upgrade the Agent.
```
